homework.md

```diff
@@ -86,19 +85,19 @@ You send the changes of your local repository to Github
 
 What is Github?
 
-A place for us to store and share our code.
+Web-based interface that uses Git and allow multiple people make changes at the same time. 
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+It is when you save changes to your local repository.
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+Microsof VS Code
```

Comment:
Hi Anderson, great work! Just noticed a small typo in the word Microsoft. Missing a 't' at the end :)

Reply:
Thank you @HamraKhan. That is fixed now!